admin/class-admin-init.php

```diff
@@ -431,6 +432,70 @@ public function unsupported_php_notice() {
 		$notification_center->remove_notification_by_id( 'wpseo-dismiss-unsupported-php' );
 	}
 
+	/**
+	 * Creates a WordPress upgrade notification in the notification center.
+	 *
+	 * @return void
+	 */
+	public function wordpress_upgrade_notice() {
+		global $wp_version;
+
+		$wordpress_less_than_50 = version_compare( $wp_version, '5.0', '<' );
+		$wordpress_less_than_52 = version_compare( $wp_version, '5.2', '<' );
+
+		$notification_center = Yoast_Notification_Center::get();
+
+		$message = sprintf(
+			/* translators: 1: opening strong tag 2: closing strong tag 3: html break 4: Yoast 5: Yoast SEO 6: 5.2 7: 5.3 */
```

Comment:
Shouldn't this be: `%1$s expands to...` etc. ??

Reply:
Changed!